benchmark/scripts/Benchmark_Driver

```diff
@@ -81,42 +81,14 @@ def submit_to_lnt(data, url):
 
 def instrument_test(driver_path, test, num_samples):
     """Run a test and instrument its peak memory use"""
-    test_outputs = []
-    for _ in range(num_samples):
-        test_output_raw = subprocess.check_output(
-            ['time', '-lp', driver_path, test],
-            stderr=subprocess.STDOUT
-        )
-        peak_memory = re.match('\s*(\d+)\s*maximum resident set size',
-                               test_output_raw.split('\n')[-15]).group(1)
-        test_outputs.append(test_output_raw.split()[1].split(',') +
-                            [peak_memory])
-
-    # Average sample results
-    num_samples_index = 2
-    min_index = 3
-    max_index = 4
-    avg_start_index = 5
-
-    # TODO: Correctly take stdev
-    avg_test_output = test_outputs[0]
-    avg_test_output[avg_start_index:] = map(int,
-                                            avg_test_output[avg_start_index:])
-    for test_output in test_outputs[1:]:
-        for i in range(avg_start_index, len(test_output)):
-            avg_test_output[i] += int(test_output[i])
-    for i in range(avg_start_index, len(avg_test_output)):
-        avg_test_output[i] = int(round(avg_test_output[i] /
-                                       float(len(test_outputs))))
-    avg_test_output[num_samples_index] = num_samples
-    avg_test_output[min_index] = min(
-        test_outputs, key=lambda x: int(x[min_index]))[min_index]
-    avg_test_output[max_index] = max(
-        test_outputs, key=lambda x: int(x[max_index]))[max_index]
-    avg_test_output = map(str, avg_test_output)
-
-    return avg_test_output
-
+    test_output_raw = subprocess.check_output(
```

Comment:
Yes, as I understand this, this just gets the max memory used by the given test. This will not be a statistical sample, just a single value. All other stats: MIN, MAX, MEAN, SD, MEDIAN are correctly  reported by `Benchmark_O` & co. and were — before this patch — dirscarded or corrupted by this script.